Kaguya/Kaguya/Discord/DiscordExtensions/TimeExtensions.cs

```diff
@@ -11,5 +12,13 @@ public static string HumanizeTraditionalReadable(this TimeSpan t)
         {
             return t.Humanize(3, minUnit: TimeUnit.Second, maxUnit: TimeUnit.Day);
         }
+
+        public static DateTime GetCurrentDaylightSavingsUtcTime()
```

Comment:
just use DateTime.UtcNow: https://docs.microsoft.com/en-us/dotnet/api/system.datetime.utcnow?view=net-5.0

Reply:
Yeah..I figured out the proper implementation of that actually. Thought something was messed up where my `DateTime.UtcNow` wasn't working, but the machine was right of course. Turns out this method isn't even used anywhere else.